packages/vm/src/environment.rs

```diff
@@ -44,8 +44,8 @@ pub struct GasConfig {
 
 impl Default for GasConfig {
     fn default() -> Self {
-        // Target is 10^12 per millisecond (see GAS.md), i.e. 10^9 gas per µ second.
-        const GAS_PER_US: u64 = 1_000_000_000;
+        // Target is 10^9 per millisecond (see GAS.md), i.e. 10^6 gas per µ second.
```

Comment:
I'd stick to "10^12 per second" for consistency here. This makes it easier to reason about.